angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts

```diff
@@ -464,7 +466,7 @@ export const extended = () => ({
             [choices]="siteDropdownChoices"
             selector="Select One"
             additionalIconClasses="sprk-c-Icon--filled-current-color sprk-Stack__item"
-            additionalTriggerTextClasses="sprk-o-Stack__item sprk-o-Stack__item--flex@xxs"
+            triggerTextAdditionalClasses="sprk-o-Stack__item sprk-o-Stack__item--flex@xxs"
```

Comment:
Let's update the rest of these input names to match the new names. 
`variant`
`triggerAdditionalClasses`
`triggerIconName`
`iconAdditionalClasses`

Reply:
There's also a `<sprk-dropdown>` on line 504. Let's update that one too.